src/components/navbar/navLinkSection/NavLinkSection.module.scss

```diff
@@ -0,0 +1,34 @@
+@import '../../../utils/variables';
+
+.link {
+    margin: 10px;
+    text-decoration: none;
+    font-weight: $semi-bold;
+    color: $blue;
+    text-align: center;
+    font-size: calc(14px + (24 - 14) * ((100vw - 320px) / (1920 - 320)));
```

Comment:
I think we should work on standardising font sizes into variables on the site to prevent variances of the `calc`s from the `Text` component being repeated around the codebase.

Reply:
Yes, I opened an issue for styling, I believe this should be included as well.
This size was carried over from previous nav which they were satisfied with, TBH I can't even read it, but It is like a basic scaling. The sizes look fair compaired to the design picture.

Reply:
Yep good point. Not in the scope of this pr but let's add a ticket to work on that!